src/pages/components/textArea/Textarea.tsx

```diff
@@ -0,0 +1,25 @@
+import { useState } from 'react';
+
+import scss from './Textarea.module.scss';
+
+interface ITextAreaProps {
+  value: string;
+}
+
+function CustomTextArea({ value }: ITextAreaProps) {
```

Comment:
Vamos usar a nomenclatura de React.FC igual o nico fez nesse PR. https://github.com/Alecell/octopost/blob/954a564ac8671a1aa518cf334081838ea066129e/src/pages/components/CharacterLimit/CharacterLimit.tsx

E também evitar desestruturar as props

```suggestion
function CustomTextArea(props): React.FC {
```